content/headless/chunk/option-destination-flags.md

```diff
@@ -6,12 +6,31 @@
 
 |          |                                |
 | -------- | ------------------------------ |
-| Type:    | no-multi-line, syslog-protocol |
+| Type:    | `ignore-errors`, `no-multi-line`, `no-seqnum-all`, `seqnum-all`, `syslog-protocol`, `threaded` |
 | Default: | empty set                      |
 
+*Description:* Flags influence the behavior of the destination driver.
 
+- `ignore-errors`: <!-- FIXME -->
```

Comment:
The `ignore-error` flag silently drops messages which have not been successfully sent out because of unrecoverable low-level errors (e.g. I/O errors).